docs/kustomization.yaml

```diff
@@ -136,3 +136,43 @@ patches:
 crds:
 - crds/typeA.yaml
 - crds/typeB.yaml
+
+# Vars are used to insert values from resources that cannot be referenced
+# otherwise. For example if you need to pass a Service's name to the arguments
+# or environment variables of a program but without hard coding the actual name
+# of the Service you'd insert `$(MY_SERVICE_NAME)` into the value field of the
```

Comment:
Can you add  a few lines showing how to insert `$(MY_SERVICE_NAME)` into container args or env